appdata.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/src/org.gnome.OfficeRunner.appdata.xml.in b/src/org.gnome.OfficeRunner.appdata.xml.in
+index 1be00fe..835f054 100644
+--- a/src/org.gnome.OfficeRunner.appdata.xml.in
++++ b/src/org.gnome.OfficeRunner.appdata.xml.in
+@@ -24,6 +24,9 @@
+   <screenshots>
+     <screenshot type="default">https://git.gnome.org/browse/office-runner/plain/src/office-runner-appdata-screenshot.jpg</screenshot>
+   </screenshots>
++  <releases>
++    <release version="1.0.3" date="2018-10-24"/>
++  </releases>
```

Comment:
This change needs to be done upstream.

Reply:
Upstream hasn't made a release in ~3years, I'll send a PR but I think the patch should be kept anyways because otherwise I need to skip the appstream check for flathub to be happy about it.

Reply:
I've made the PR, mark this as resolved if you believe that the patch should be kept here until a next release.

Reply:
This is the great thing about the manifest mentioning a specific git commit, you can change it without waiting for a new release. So you'll need 3 separate commits:
- one to change the commit we're pointing to, so it includes the appstream patch
- one to install gnome-common (make sure to include a `"cleanup": "0"` equivalent so all the gnome-common files get removed from the installed application)
- one to bump the SDK version

Let me know if any of those are unclear.

Reply:
I checked the commits, expected more of them but it ended up being rather short. There's also translation changes.